RubeesEat/Model/DB/DbBillRepository.cs

```diff
@@ -224,7 +225,8 @@ private IReadOnlyList<Bill> CreateBillsFromReader(DbDataReader reader)
             var person = new Person(
                 reader.GetGuid(5),
                 reader.GetString(6),
-                reader.GetString(7)
+                reader.GetString(7),
+                reader.IsDBNull(8) ? null : reader.GetString(8)
             );
```

Comment:
Not somehting we will change now and I don't know what input @a-ctor had about it, but generally speaking, having the SELECT projection and the column access separated between methods isn't good since it means the information about the column order is lost in the call chain. It is possible that there's no good alternative without using an O/R-mapper.